backend/game_server/lib/helpers.ex

```diff
@@ -0,0 +1,2 @@
+defmodule BernacleServer.Helpers do
+end
```

Comment:
:D I believe you will be better in this so fast that this shortcuts won't be necessery. If you use sublime look at `ensure_newline_at_eof_on_save` option ;)